README.md

```diff
@@ -56,6 +56,11 @@ If you would like to add a resource or contribute in any other kind of way, be s
   - [IRC / Slack support](#irc--slack-support)
 - [Contributing to Drupal](#contributing-to-drupal)
 - [Video tutorials](#video-tutorials)
+  - [Beginner Training Series](#beginner-training-series)
+  - [General](#general)
+  - [Distributions](#distributions)
+  - [Modules](#modules)
```

Comment:
Since there is already a `modules` section, we need to update the anchor to `#modules-1`